code-review-tester-service/Program.cs

```diff
@@ -1,6 +1,42 @@
+using Microsoft.AspNetCore.Mvc;
+
 var builder = WebApplication.CreateBuilder(args);
 var app = builder.Build();
 
 app.MapGet("/", () => "I'm ALIVE!");
 
+app.MapGet("/tests/", (string? level) => 
+{
+    // If no level is provided, return all available levels
+    if (string.IsNullOrEmpty(level))
+    {
+        return Results.Ok(new[] { "Easy", "Medium", "Hard", "Extra Hard" });
+    }
+
+    // Validate the level parameter
+    if (!Enum.TryParse<DifficultyLevel>(level.Replace(" ", ""), true, out var parsedLevel))
+    {
+        return Results.BadRequest($"Invalid level '{level}'. Valid levels are: Easy, Medium, Hard, Extra Hard");
```

Comment:
This should also just list the items from the enum rather than retyping them here.